src/components/FontReport.vue

```diff
@@ -33,5 +33,10 @@ export default {
 		StyleSheet,
 		SiteFooter,
 	},
+	updated() {
+		if (this.font) {
+			document.getElementById("report")?.scrollIntoView();
```

Comment:
I'd need to retrain my muscle memory for the `?`, I unlearned it when moving from TypeScript to regular old JavaScript, which isn't necessary!

Reply:
You can think of it (in this particular case) as shorthand for

```js
document.getElementById("report") != undefined
  ? document.getElementById("report").scrollIntoView()
  : undefined;
```

(Also https://babeljs.io/repl/ might help?)